openassessment/templates/openassessmentblock/oa_team_uploaded_files.html

```diff
@@ -0,0 +1,38 @@
+{% spaceless %}
+{% load i18n %}
+
+{% if file_upload_type %}
+  <div class="{{ class_prefix }}__display__file {% if not team_file_urls %}is--hidden{% endif %} submission__{{ file_upload_type }}__upload" data-upload-type="{{ file_upload_type }}">
+        {% if team_file_urls %}
+        <h5 class="submission__answer__part__text__title">
+          {% trans "Files that were uploaded by your teammates:" %}
+        </h5>
+        {% endif %}
+
+        <div class="submission__answer__files">
+        {% for file_url, file_description, file_name, show_delete_button in team_file_urls %}
```

Comment:
Just checking: since these files are team-mate owned, the delete button will never be displayed?
Also, I may have missed, but would it make sense to have a unit tests that checks for correct rendering of the page?

Reply:
yeah, exactly, you can't delete files that someone else is the owner/original-uploader of